### ISSUE TITLE
ITIS: two species in tribe Agabini have tribe as a parent

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/data#232]
Dear @DaveNicolson, my apology, fixing one problem we met another one.

We have got split genus Agabus in CoL tree like that: 
![image](https://user-images.githubusercontent.com/28574252/105423735-b706bd80-5c0b-11eb-9e17-1936dd5d6992.png)

After our with @gdower investigation, we understood that 2 species _Agabus terminalis_ and _Agabus valdiviensis_ have tribe Agabini as direct parent. These species are not assigned in the genus Agabus.

https://itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=112060#null
https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=811471#null

Is this correct?
How to manage such cases in the CoL?

**Comment by DaveNicolson** [comment]
These are two species that were determined to not belong to the genus Abagus, but they were not synonymized. As such, we put them at the level just above the genus (so the tribe), and ticked off a box that forces them to be listed as incertae sedis therein. See here: https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=728216#null

I don't know what your options are....

**Comment by yroskov** [comment]
@chantalhuijbers This is a question to Taxonomy Group: How to manage such cases in the CoL?

**Comment by yroskov** [comment]
Another example: **14 species from two amphibian genera have subfamily as as a direct parent**.

Genus Centrolene. Genus Not assigned: 7 spp with Subfamily Centroleninae as a direct parent
Genus Cochranella. Genus Not assigned: 7 spp with Subfamily Centroleninae as a direct parent

Original view in ITIS. Species with Uncertain Position: 14 spp have subfamily as as a direct parent
https://itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=773201#null

![image](https://user-images.githubusercontent.com/28574252/105897492-dae84b80-5fdd-11eb-8349-b90750db8327.png)

**Comment by DaveNicolson** [comment]
Yes, those species were all listed directly under the subfamily in Amphibian Species of the World when we last updated about a year or so ago. In ASW they put quotes around the genus name to indicate that they were published in those genera but were no longer understood to belong there. I think one of them has been placed in synonymy under another species since then, but the others are still there in ASW, so this issue is not going away anytime soon...

**Comment by yroskov** [comment]
Yes, I see a reason why such cases happened in ITIS. 

In the CoL, we have detected 20 genera from ITIS which became split between different branches of the Tree for that reason. ITIS is not unique, few other GSDs do the same, but in different formats.

The question for Taxonomy Group: how to manage such cases in CoL:
1. Leave split genera as they are (i.e. as data came from source checklists)
2. Modify software to change presentation (how such cases should appear in the Tree?)
3. Bring "orphan" set of species back to the location of original genus through editorial decision in CoL. Huge work. Not sure, how sustainable such modifications can be in the current software through updates with changing classifications.

**Comment by mdoering** [comment]
I very much like to see a recommendation from the Taxonomy Group on this.
Not so much how the software should deal with it, but rather how to deal with this in an ideal way much along those 3 options @yroskov has already listed.

As a long time user (GBIF) of the COL I always found the existance of "split" genera in COL wrong and considered it an error. I do understand now there are good reasons for this, but it is not at all obvious to users. It just seems like an error. If COL should stick with "split" genera then I think we need to mark them clearly so users can cleanly separate these cases from actual duplicates or errors.
